index.bs

```diff
@@ -1304,6 +1315,17 @@ controls the maximum [=map/size=] of a [=trigger spec map=] for an
 the maximum [=string/length=] of an [=attribution trigger=]'s [=attribution trigger/trigger context ID=].
 Its value is 64.
 
+<dfn>Default max event states</dfn> is a positive integer that controls the
+default [=attribution source/max event states=]. Its value is 3.
+
+<dfn>Max length per attribution scope for source</dfn> is a positive integer that controls the
```

Comment:
Nitpick -- can we have language that states what is the length "of"?

Like just above:
```
<dfn>Max length per trigger context ID</dfn> is a positive integer that controls
the maximum [=string/length=] of an [=attribution trigger=]'s [=attribution trigger/trigger context ID=].
Its value is 64.
```

Reply:
Oh -- oops -- I see the naming is similar. Never mind! (Cannot resolve comment)

Reply:
Seems odd to me to say "per" rather than "of."

Reply:
Done.